CRM/Report/Form.php

```diff
@@ -1155,8 +1155,10 @@ public function cleanUpTemporaryTables() {
    * @return string
    */
   public function createTemporaryTable($identifier, $sql, $isTrueTemporary = TRUE) {
+    $name = CRM_Utils_SQL_TempTable::build()->setUtf8(TRUE)->setDurable(($isTrueTemporary !== TRUE))->createWithQuery($sql)->getName();
```

Comment:
@stesi561 hmm - so I guess this isn't quite right - as in 

when we call build it sets 
    $t->durable = CRM_Utils_Constant::value('CIVICRM_TEMP_FORCE_DURABLE', FALSE);

In fact we don't want to override that UNLESS we are trying to force it to not be a temporary table (which used to be pretty common for debuggers to do but less so now we can turn it on & off with a constant. 

So I think I need to tweak this - just figuring out how....